src/gui/uithememanager.cpp

```diff
@@ -58,6 +61,115 @@ namespace
 
         return {};
     }
+
+    class QRCThemeSource : public UIThemeSource
+    {
+        Q_DECLARE_TR_FUNCTIONS(QRCThemeSource)
```

Comment:
IMO, it isn't a good idea to create yet another translation context for such "internal" classes. It would be better either translate the strings using `UIThemeManager` context (i.e. `UIThemeManager::tr("Some text")`) or just indicate error to caller side (e.g. by returning some kind of `nonstd::expected`) and produce log messages there.